src/java/htsjdk/samtools/CRAMIterator.java

```diff
@@ -101,10 +101,10 @@ public CRAMIterator(final InputStream inputStream, final ReferenceSource referen
     public CRAMIterator(final SeekableStream seekableStream, final ReferenceSource referenceSource, final long[] coordinates)
             throws IOException {
         if (null == referenceSource) {
-            throw new CRAMException("A reference source is required for CRAM files");
-        }
+            this.referenceSource = new ReferenceSource();
+        } else
+            this.referenceSource = referenceSource;
```

Comment:
Same note above about missing curly braces and relying on a a default-constructed `ReferenceSource` applies here as well.